src/package/pywinrt/projection/generate.ps1

```diff
@@ -0,0 +1,43 @@
+param ([switch]$clean, [switch]$fullProjection)
+
+$windows_sdk = '10.0.17763.0'
+$repoRootPath = (get-item $PSScriptRoot).parent.Parent.parent.Parent.FullName
```

Comment:
>.parent.Parent.parent.Parent [](start = 40, length = 28)

camel parenting, nice

Reply:
these scripts are only for local use, so I'm not going to worry about this casing thing

Reply:
I thought it was intentional, to keep track of the number of them

---
In reply to: [278704485](https://github.com/Microsoft/xlang/pull/356#discussion_r278704485) [](ancestors = 278704485)

Reply:
Uhh, yeah that's it! It was intentional to keep track of the number of them!